installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc

```diff
@@ -5,10 +5,38 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following troubleshooting sections address Installer Provisioned Infrastructure (IPI) troubleshooting.
 
-// For general, {product-title} troubleshooting, see <xref>.
+.Troubleshooting the installer workflow
```

Comment:
To ensure this is area of content is included in the TOC, we should use this heading format instead:

= Troubleshooting the installer workflow